websockets/handlers/echo-cookie_wsh.py

```diff
@@ -3,7 +3,7 @@
 from mod_pywebsocket import msgutil
 
 def web_socket_do_extra_handshake(request):
-    request.ws_cookie = request.headers_in.get('Cookie')
+    request.ws_cookie = request.headers_in.get('Cookie') or request.headers_in.get('cookie')
```

Comment:
It seems weird that you need to do this. I file https://github.com/GoogleChromeLabs/pywebsocket3/issues/18 but I don't know whether we'll fix it there.

Reply:
in the case of ws over h2, headers_in are of type H2Headers https://github.com/web-platform-tests/wpt/blob/16d86f34f5b91f9588148fe16624f026bb25430d/tools/wptserve/wptserve/server.py#L615

The alternative would be to modify H2Headers to match fields case-insensitively, though I'm worried of breaking other tests.

Reply:
I see, thanks. Maybe experiment with changing H2Headers in a separate CL?

Reply:
Actually, I think we can get away with only the lower case versions, since if we are doing a case-sensitive comparison, we must be using H2.

Reply:
Good idea. I have uploaded a new patch.